pom.xml

```diff
@@ -151,7 +151,7 @@
         <jax-rs-api.version>2.0.1</jax-rs-api.version>
         <maxmind.version>0.8.1</maxmind.version>
         <jna.version>4.0.0</jna.version>
-        <winrm4j.version>0.3.1</winrm4j.version>
+        <winrm4j.version>0.4.0</winrm4j.version>
```

Comment:
winrm4j version `0.3.2` is on its way to maven central - can you update to use that instead please?